src/components/browser/__tests__/browser-test-cases.js

```diff
@@ -0,0 +1,41 @@
+import Browser from '../browser';
+import React from 'react';
+
+const testCases = {};
+
+testCases.basic = {
+  component: Browser,
+  description: '',
+  props: {
+    children: (
+      <img
+        src="https://docs.mapbox.com/studio-manual/assets/ref-styles-manage-styles-1200-599fd585f5f2d8e0729d5fe37f9722c0.webp"
```

Comment:
This image is not loading for me in Safari.

![image](https://user-images.githubusercontent.com/10479155/63984699-447ac800-ca81-11e9-8715-2dd17701107d.png)

Reply:
Ohh is that the example image is broken? I'm using a webp image, I should use a png instead!

Reply:
![image](https://user-images.githubusercontent.com/2180540/64025746-66209180-cb0b-11e9-956c-78bcd7b53980.png)

Fixed! I added an `assets` folder in `src/tests-cases-app/` so we can store images/videos/etc that will be bundled only for the test cases site.

Reply:
oooo yes!